Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,203 @@
+# Team 4 – Mario Kart & Ride Replay Simulator
+
+## Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham ##
```

Comment:
Maybe reduce this from a header to just text

Reply:
Done; assuming you were referring to the member listing and not the title.